lib/map.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require('react');
+
+var MapComponent = React.createClass({
```

Comment:
Can we change `MapComponent` to `SingletonMapComponent` here too just so it's clear to folks who are reading this file?